index.js

```diff
@@ -8,8 +8,51 @@ const http = require('http'),
         The time is: ${rightNow}
       </p>
     </body>
+ 
+      <body>
+   <div style="position:  absolute; bottom: 5px; background-color: white">
```

Comment:
Is there are better/correct-err(?) way to anchor the character list to the bottom of the page?